paasta_tools/cli/cmds/status.py

```diff
@@ -602,6 +602,36 @@ def format_marathon_task_table(tasks):
     return format_table(rows)
 
 
+def format_kubernetes_pod_table(pods):
+    rows = [("Pod ID", "Host deployed to", "Deployed at what localtime", "Health")]
+    for pod in pods:
+        local_deployed_datetime = datetime_from_utc_to_local(
+            datetime.fromtimestamp(pod.deployed_timestamp)
+        )
+        hostname = f"{pod.host}" if pod.host is not None else "Unknown"
+
+        if pod.phase is None:
+            health_check_status = PaastaColors.grey("N/A")
+        elif pod.phase == "Running":
```

Comment:
I'm not sure if this is a good indication of healthness of pods. Running does not necessarily mean the pod is healthy . It could transition to failed state after running for a few seconds. And pending does not mean it's unhealthy. Perhaps we should use liveness probe instead.

Reply:
My personally option is that pending should be considered as unhealthy. I think if liveness check fails, the phase would transit to failed state, so the phase is actually taking liveness probe into account. Besides, the liveness_probe doesn't contain the result of liveness check:
                          'liveness_probe': {'_exec': None,
                                             'failure_threshold': 30,
                                             'http_get': {'host': None,
                                                          'http_headers': None,
                                                          'path': '/status',
                                                          'port': 8888,
                                                          'scheme': 'HTTP'},
                                             'initial_delay_seconds': 60,
                                             'period_seconds': 10,
                                             'success_threshold': 1,
                                             'tcp_socket': None,
                                             'timeout_seconds': 10}, 

I think eventually the healthness should be determined by the smartstack (I plan to add smartstack section through another PR).

Reply:
@chlgit Why can't just make pending pending? And unhealthy if it's failed or is running but unhealthy? I would assume pending != unhealthy if I were the user.

Reply:
@chlgit  Or we can even have both phase and health_check_status. health_check_status is unknown unless we can really make sure we know it.